packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html

```diff
@@ -128,7 +128,7 @@ <h2>
         <oui-step-form
             data-header="{{:: 'pci_projects_project_training_jobs_submit_select_data' | translate  }}"
             data-editable="!$ctrl.isSubmit"
-            data-on-submit="$ctrl.forceSubmitContainer(form)"
+            data-on-submit="form.$valid && $ctrl.forceSubmitContainer(form)"
```

Comment:
You could use `prevent-next` + `valid` to make validation.
(See https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-stepper-webcomponent--disable-next-button)